observer/bin/spark-observer.js

```diff
@@ -47,7 +43,14 @@ const loop = async (name, fn, interval) => {
     }
     const dt = Date.now() - start
     console.log(`Loop "${name}" took ${dt}ms`)
-    await timers.setTimeout(interval - dt)
+    recordTelemetry(`loop_${name.replaceAll(' ', '_')}`, point => {
+      point.intField('interval_ms', interval)
+      point.intField('duration_ms', dt)
+      point.intField('delay_ms', interval - dt)
+    })
```

Comment:
I am envisioning an alert like "if delay_ms/interval_ms is less than 10%"

Reply:
Well, maybe "if delay_ms/interval_ms is less than 40%" is better as it will give us more time to fix the issue before it becomes a problem.